src/test/java/org/example/c20debugging/UtilitiesTestParameterized.java

```diff
@@ -0,0 +1,41 @@
+package org.example.c20debugging;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import static org.junit.jupiter.api.Assertions.*;
+import java.util.Arrays;
+import java.util.Collection;
+
+@RunWith(Parameterized.class)
+public class UtilitiesTestParameterized {
+
+    private Utilities util;
+
+    private String input;
+    private String output;
+
+    public UtilitiesTestParameterized(String input, String output) {
+        this.input = input;
+        this.output = output;
+    }
+
+    @org.junit.Before
```

Comment:
Use `org.junit.jupiter` imports instead of `org.junit`

Reply:
done